src/_shared/js/utils.js

```diff
@@ -0,0 +1,8 @@
+
```

Comment:
Oh, I forgot about this one. Yeah slowly but surely the `js` and `scss` folders are getting unwieldy, let's think about this in a future comprehensive PR.